.github/workflows/python-app.yml

```diff
@@ -53,27 +53,34 @@ jobs:
     - name: djangosaml2 SP metadata to Proxy
       run: |
         wget http://localhost:8000/saml2/metadata -O example/metadata/sp/djangosaml2_sp.xml
+    - name: Create docker-example folder
+      run: |
+        mkdir -p docker-example
+        cp -r example/. docker-example
     - name: run satosa-saml2spid
       run: |
-        cd example
+        cd docker-example
         mkdir -p metadata/idp
         mkdir -p metadata/sp
         export SATOSA_APP=`python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])'`
```

Comment:
this is not needed anymore, please remove

Reply:
did you mean remove "cd docker-example" ?

Reply:
no, exactly the line above my comment

`export SATOSA_APP=`python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])'``